index.html

```diff
@@ -1157,10 +1156,22 @@ <h3 id="applying">
           Applying the manifest
         </h3>
         <p>
-          A <a>manifest</a> is <dfn data-lt="apply|applying">applied</dfn> to a
-          <a>top-level browsing context</a>, meaning that the members of the
-          <a>manifest</a> are affecting the presentation or behavior of a
-          browsing context.
+          A <a>manifest</a> can be <dfn data-lt="apply|applying">applied</dfn>
+          to a <a>top-level browsing context</a>, meaning that the members of
+          the <a>manifest</a> are affecting the presentation or behavior of a
+          browsing context. Whenever a <a>top-level browsing context</a> is
+          created, the user agent MAY <a>apply</a> a manifest to it, before
+          [=navigate|navigation=] begins.
+        </p>
+        <p class="note">
+          Whether to [=apply=] a manifest, and which manifest to apply, is at
+          the discretion of the user agent, based on the user's actions. For
+          example, if the user launched an application from the system menu or
+          from a [=launching a shortcut|shortcut=], the user agent might create
+          a new [=top-level browsing context=] with that application's
+          [=manifest=] [=applied=], but it might not do so if the user simply
+          clicked a bookmark to a URL within the application's [=navigation
+          scope=].
```

Comment:
so you mean that it might reuse an existing one?

Reply:
This is meant to imply that it might either reuse an existing one (navigate the user's current tab) or open a new browser tab, as opposed to creating an app window.

The point is that the browser can decide based on the user's action whether to open an app window or a browser tab. (This is really just because the current spec says "a manifest is applied" but it doesn't say if it's always applied, or when exactly a user agent should apply a manifest. I'm just being explicit and saying "the user agent can do that at the appropriate time, but doesn't have to".)

Reply:
... or could even apply a user configured one...

Reply:
Do you think it's worth pointing that out, Marcos? Seems like a kind of advanced use case that would be allowed, but not sure we need to call it out here.

Reply:
sorry, no need to call that out... was just an example for @kenchris.